### ISSUE TITLE
Three collision roll for damage when bumping friendly ship

### ISSUE THREAD

**Issue by Korran12** [Sandrem/FlyCasual#3437]
![Captura de pantalla 2022-02-25 a la(s) 8 49 44 p  m](https://user-images.githubusercontent.com/67489605/155826147-99885c74-3123-41cc-ad03-655eaec11a43.png)
FlyCasual for MacOS Version: 2.0

**Comment by Korran12** [comment]
Yeah, I don't remember how many previous matches I had before but this wasn't the first one for sure.

**Comment by emperorscanaries** [comment]
> Was it the first match after launch of the game, or did you play ~3 matches in a row?

I had this same problem. I believe my first game worked fine, the second game I exited right away because I had a wrong pilot in my list, then the problem occurred on my third game. 

Additionally, on that third game, bumping ships also created 3 red focus triggers.

**Comment by grandpajive** [comment]
Yes, it seems the best workaround is to close and re-open for now.

**Comment by MichalRybinski** [comment]
![image](https://user-images.githubusercontent.com/4457854/197885619-f92d388a-2e14-4e5f-bc5f-d4a23313df3c.png)

Happened for every 'Restart Game'
